conf/configuration.go

```diff
@@ -275,6 +276,7 @@ func init() {
 	viper.SetDefault("loglevel", "info")
 	viper.SetDefault("address", "0.0.0.0")
 	viper.SetDefault("port", 4533)
+	viper.SetDefault("unixsocketperm", 0017)
```

Comment:
Here you are mixing permissions with umask. We want the default to be something like `0660`. A permission of `0017` would not allow processes from the same user to open to the socket.